packages/ui-tree-browser/src/TreeBrowser/__tests__/TreeBrowser.test.tsx

```diff
@@ -909,4 +909,63 @@ describe('<TreeBrowser />', async () => {
       expect((await tree.findAllItems()).length).to.equal(4)
     })
   })
+
+  describe('sorting', async () => {
```

Comment:
It is nice that you wrote a test for it! Please add an example to the docs page as well, so that ppl can see how it works. Thanks!